inhibit/inhibit.go

```diff
@@ -72,9 +71,7 @@ func (ih *Inhibitor) run(ctx context.Context) {
 			// Update the inhibition rules' cache.
 			for _, r := range ih.rules {
 				if r.SourceMatchers.Matches(a.Labels) {
```

Comment:
nit: maybe this could go inside the `set` method because it is easier to understand why the flag `matchesSourceAndTarget` is being set to `true` this way.